src/calendar-heatmap.js

```diff
@@ -111,31 +141,78 @@ function calendarHeatmap() {
         });
       }
 
+      if (dayNumbersInBox){
+        dayNumbers = svg.selectAll('.day-numbers')
+        .data(dateRange);  //  array of days for the last yr
+        
+        dayNumbers.enter().append('text')
+        .attr('class', 'day-numbers')
+        .attr("font-size", dayNumbersFontSz+"px")
```

Comment:
Just a code style thing - we use single quotes here and put spaces either side of the `+`